scripts/cumberbatch.coffee

```diff
@@ -0,0 +1,19 @@
+# Description:
+#   Benedict Cumberbatch name generator.
+#
+# Dependencies:
+#   None
+#
+# Configuration:
+#   None
+#
+# Commands:
+#   cumberbatch - Creates your very own Benedict Cumberbatch
+#
+# Author:
+#   mbulmer
+
+module.exports = (robot) ->
+  robot.respond /(cumberbatch)/i, (msg) ->
```

Comment:
I wouldn't say anything we do is **appropriate** but the robot would fire the script whenever it hears anyone say `/(cumberbatch)/i` regardless of being called directly. 

See: https://github.com/github/hubot/blob/master/docs/scripting.md#hearing-and-responding